source/includes/gcp/_target_proxies.md

```diff
@@ -84,13 +89,62 @@ Retrieve a target proxy in an [environment](#administration-environments).
 | -------------------------------- | ------ |
 | `creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format. |
 | `id`<br/>*UUID*                  | The id of the backend service. |
-| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies.|
+| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies. |
 | `name`<br/>*string*              | Name of the resource. |
 | `selfLink`<br/>*string*          | Server-defined URL for the resource. |
 | `shortUrlMap`<br/>*string*       | A short version of the URL map. |
-| `type`<br/>*enum*                | A short version of the kind of resource. Either HTTP, HTTPS, SSL or TCP|
+| `shortCertificates`<br/>*string* | A short version of the SSL certificates. |
+| `type`<br/>*enum*                | A short version of the kind of resource. Either HTTP, HTTPS, SSL or TCP. |
 | `urlMap`<br/>*string*            | URL to the UrlMap resource that defines the mapping from URL to the BackendService. |
 
+<!-------------------- CREATE A TARGET PROXY -------------------->
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/targetProxies"
+
+# Examples:
+```
+
+```json
+{
+   "name": "targetProxyName",
+   "type": "HTTPS",
+   "shortUrlMap": "urlMapName",
+   "shortCertificates": "sslCertificateName"
+}
+```
+
+```json
+{
+   "name": "targetProxyName",
+   "type": "HTTPS",
+   "shortBackend": "backendServicesName",
+   "shortCertificates": "sslCertificateName"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/targetProxies</code>
+
+Create a new target proxy
+
+| Attributes | &nbsp;|
```

Comment:
Which ones are required ? which ones optional ?

Reply:
An example: 
![image](https://user-images.githubusercontent.com/34725714/66858051-8c34b080-ef56-11e9-9a0e-1aa17799e289.png)

Reply:
The API requires either a backend service or a URL map. One of them is required, but not both.

Reply:
But them in Required, mentioning it's one or the other. 'Mutually exclusive'

Reply:
ae81aa9d03315427864b7953c7932bce5c8f5420